module/entities/helper.js

```diff
@@ -0,0 +1,208 @@
+/**
+ * name : helper.js
+ * author : Ankit Shahu
+ * created-date : 16-Jan-2024
+ * Description : entities helper functionality.
+ */
+
+const userService = require(GENERICS_FILES_PATH + "/services/users");
+
+module.exports = class entitieHelper {
+
+ /**
+   * update registry in entities.
+   * @method
+   * @name listByLocationIds
+   * @param {Object} locationIds - locationIds
+   * @returns {Object} entity Document
+   */
+
+    static listByLocationIds(locationIds) {
+        return new Promise(async (resolve, reject) => {
+            try {
+                //if not uuid considering as location code- for school.
+                let locationDeatails = UTILS.filterLocationIdandCode(locationIds);
+                //set request body for learners api
+                let entityInformation = [];
+                let formatResult = true;
+
+                if ( locationDeatails.ids.length > 0 ) {
+                    let bodyData = {
+                        "id" : locationDeatails.ids
+                    } 
+                    let entityData = await userService.locationSearch( bodyData, formatResult );
+                    if ( entityData.success ) {
+                        entityInformation =  entityData.data;
+                    }
+                }
+
+                if ( locationDeatails.codes.length > 0 ) {
+                    let bodyData = {
+                        "code" : locationDeatails.codes
+                    } 
+                    let entityData = await userService.locationSearch( bodyData, formatResult );
+                    if ( entityData.success ) {
+                        entityInformation =  entityInformation.concat(entityData.data);
+                    }
+                }
+
+                if ( !entityInformation.length > 0 ) {
+                    throw {
+                        message : CONSTANTS.apiResponses.ENTITY_NOT_FOUND
+                    } 
+                }
+
+                return resolve({
+                    success : true,
+                    message : CONSTANTS.apiResponses.ENTITY_FETCHED,
+                    data : entityInformation
+                });
+
+            } catch(error) {
+                return resolve({
+                    success : false,
+                    message : error.message
+                });
+            }
+        })
+    }
+
+
+      /**
+     * Get Observation document based on filtered data provided.
+     * @method
+     * @name observationDocuments
+     * @param {Object} [findQuery = "all"] -filter data.
+     * @param {Array} [fields = "all"] - Projected fields.
+     * @returns {Array} - List of observations.
+     */
+
+      static observationDocuments(findQuery = "all", fields = "all") {
```

Comment:
@Prajwal17Tunerlabs why observation document in entities collection. Please create new databaseQueries observation and add that there